cli/commands/flags.go

```diff
@@ -172,6 +172,9 @@ const (
 	TerragruntProviderCacheRegistryNamesFlagName = "terragrunt-provider-cache-registry-names"
 	TerragruntProviderCacheRegistryNamesEnvName  = "TERRAGRUNT_PROVIDER_CACHE_REGISTRY_NAMES"
 
+	TerragruntFeatureMapFlagName = "feature"
+	TerragruntFeatureMapEnvName  = "TG_FLAG"
```

Comment:
I know I used `TG_FLAG` in the RFC, but I think we should stick to `TERRAGRUNT_FLAG` here.

My reasoning was that we were going to use `TG_FLAG_` as the prefix for the flag name, but given that we're just using a key-value pair in the value separated by `=`, I think we can use the standard name.